<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendBox</class>
 <widget class="QDialog" name="SendBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>291</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>292</width>
    <height>291</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>292</width>
    <height>291</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ارسال</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/files/icons8_change_48px.png</normalon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.739, y2:0.71, stop:0 rgba(0, 167, 255, 255), stop:1 rgba(117, 255, 255, 255));</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:1 rgba(0, 0, 0, 10));</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string>background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.534, y2:0.171, stop:0.255682 rgba(0, 0, 0, 0), stop:0.420455 rgba(0, 0, 0, 0), stop:0.744318 rgba(0, 0, 0, 0), stop:0.994318 rgba(235, 148, 61, 0));</string>
    </property>
    <property name="text">
     <string>دسته مورد نظر را انتخاب کنید!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="select">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.534, y2:0.171, stop:0.255682 rgba(0, 0, 0, 0), stop:0.420455 rgba(0, 0, 0, 0), stop:0.744318 rgba(0, 0, 0, 0), stop:0.994318 rgba(235, 148, 61, 0));</string>
    </property>
    <property name="text">
     <string>ارسال</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/files/icons8_tick_box_100px.png</normaloff>:/files/icons8_tick_box_100px.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.534, y2:0.171, stop:0.255682 rgba(0, 0, 0, 0), stop:0.420455 rgba(0, 0, 0, 0), stop:0.744318 rgba(0, 0, 0, 0), stop:0.994318 rgba(235, 148, 61, 0));</string>
    </property>
    <property name="text">
     <string>بازگشت</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/files/icons8_go_back_100px.png</normaloff>:/files/icons8_go_back_100px.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>51</y>
      <width>251</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="empUsername_support">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="support">
         <property name="font">
          <font>
           <family>B Nazanin</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string>background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.534, y2:0.171, stop:0.255682 rgba(0, 0, 0, 0), stop:0.420455 rgba(0, 0, 0, 0), stop:0.744318 rgba(0, 0, 0, 0), stop:0.994318 rgba(235, 148, 61, 0));</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="markting_2">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string>background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.534, y2:0.171, stop:0.255682 rgba(0, 0, 0, 0), stop:0.420455 rgba(0, 0, 0, 0), stop:0.744318 rgba(0, 0, 0, 0), stop:0.994318 rgba(235, 148, 61, 0));</string>
         </property>
         <property name="text">
          <string>پشتیبانی:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QComboBox" name="empUsername_marketing">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="marketing">
         <property name="font">
          <font>
           <family>B Nazanin</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string>background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.534, y2:0.171, stop:0.255682 rgba(0, 0, 0, 0), stop:0.420455 rgba(0, 0, 0, 0), stop:0.744318 rgba(0, 0, 0, 0), stop:0.994318 rgba(235, 148, 61, 0));</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="markting">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string>background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.534, y2:0.171, stop:0.255682 rgba(0, 0, 0, 0), stop:0.420455 rgba(0, 0, 0, 0), stop:0.744318 rgba(0, 0, 0, 0), stop:0.994318 rgba(235, 148, 61, 0));</string>
         </property>
         <property name="text">
          <string>مارکتینگ:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QComboBox" name="empUsername_operator">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="operator_2">
         <property name="font">
          <font>
           <family>B Nazanin</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string>background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.534, y2:0.171, stop:0.255682 rgba(0, 0, 0, 0), stop:0.420455 rgba(0, 0, 0, 0), stop:0.744318 rgba(0, 0, 0, 0), stop:0.994318 rgba(235, 148, 61, 0));</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Opratorss">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string>background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.534, y2:0.171, stop:0.255682 rgba(0, 0, 0, 0), stop:0.420455 rgba(0, 0, 0, 0), stop:0.744318 rgba(0, 0, 0, 0), stop:0.994318 rgba(235, 148, 61, 0));</string>
         </property>
         <property name="text">
          <string>اپراتور:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="boss">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.534, y2:0.171, stop:0.255682 rgba(0, 0, 0, 0), stop:0.420455 rgba(0, 0, 0, 0), stop:0.744318 rgba(0, 0, 0, 0), stop:0.994318 rgba(235, 148, 61, 0));</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRepeatDelay">
          <number>300</number>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="markting_3">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string>background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.534, y2:0.171, stop:0.255682 rgba(0, 0, 0, 0), stop:0.420455 rgba(0, 0, 0, 0), stop:0.744318 rgba(0, 0, 0, 0), stop:0.994318 rgba(235, 148, 61, 0));</string>
         </property>
         <property name="text">
          <string>مدیر:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="notif">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string>background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.534, y2:0.171, stop:0.255682 rgba(0, 0, 0, 0), stop:0.420455 rgba(0, 0, 0, 0), stop:0.744318 rgba(0, 0, 0, 0), stop:0.994318 rgba(235, 148, 61, 0));</string>
    </property>
    <property name="text">
     <string>نوتفیکیشن</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
